docs/source/models.rst

```diff
@@ -266,52 +266,58 @@ For more info, see:
     mlflow pyfunc serve --help
     mlflow pyfunc predict --help
 
-Deploy a ``python_function`` model on Microsoft AzureML
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Microsoft Azure ML
+^^^^^^^^^^^^^^^^^
+The :py:mod:`mlflow.azureml` module can package ``python_function`` models into Azure ML container images.
+These images can be deployed to Azure Kubernetes Service (AKS) and the Azure Container Instances (ACI)
+platform for real-time serving.
+
+* :py:func:`build_image <mlflow.azureml.build_image>` registers an MLflow model with an existing Azure ML
```

Comment:
Cross-ref here also doesn't work, would be good

Reply:
The cross-ref seems to work when the docs are built with Python 3. Previously, if the docs were built with Python 2, the Azure ML docs would not be generated because `mlflow.azureml` attempts to import from the SDK, which only supports Python3; this import error caused this portion of the docs to be skipped. To fix this problem, I removed these SDK imports from the top-level module and moved them into `build_image` - you should be able to build the docs successfully using both major versions of python.